sklearn/model_selection/tests/test_validation.py

```diff
@@ -61,6 +61,7 @@
 from sklearn.datasets import make_multilabel_classification
 
 from sklearn.model_selection.tests.test_split import MockClassifier
```

Comment:
There are quite a few `MockClassifier`'s that are specific to the tests and the module. I'm not sure there is much benefit in grouping them into `utils.testing`...